de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/nodes/RJsonHandler.java

```diff
@@ -87,15 +88,22 @@ public void loadParametersIntoWorkspace(String parameterJson, String sourceParam
     String language = parameterData.getGeneratorLanguage();
     for (DataArray param : parameterData.getParameters()) {
       if (sourceParam.equals(param.getMetadata().getId())) {
+        // store data in temp file because moving big arrays between controller and Java
+        // doesn't seem to work properly
+        File temp_data = FileUtil.createTempFile("data", "json");

```

Comment:
Local variable and method parameter names should comply with a naming convention (java:S117).

Shared naming conventions allow teams to collaborate effectively. This rule raises an issue when a local variable or function parameter name does not match the provided regular expression.

Noncompliant Code Example

With the default regular expression ^[a-z][a-zA-Z0-9]*$: